flink-connector-examples/src/main/resources/log4j.properties

```diff
@@ -16,3 +16,6 @@ log4j.appender.console.layout=org.apache.log4j.PatternLayout
 log4j.appender.console.layout.ConversionPattern=%d{HH:mm:ss,SSS} %-5p %-60c %x - %m%n
 
 log4j.logger.io.pravega.end2end=INFO
+log4j.logger.io.pravega.client.stream.impl.ControllerImpl=OFF
```

Comment:
Is this disabled because the output is too verbose for the user executing the app?

Reply:
yes